features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -2541,6 +2541,12 @@ CREATE INDEX IDN_SECRET_TYPE_ID ON IDN_SECRET (TYPE_ID);
 CREATE INDEX IDX_CLAIM_TI_CU ON IDN_CLAIM (TENANT_ID, CLAIM_URI)
 /
 
+-- IDP_AUTHENTICATOR --
+CREATE INDEX IDX_AUTH_DEFINED_BY ON IDP_AUTHENTICATOR (DEFINED_BY)
```

Comment:
```suggestion
CREATE INDEX IDX_AUTH_DEFINED_BY ON IDP_AUTHENTICATOR (DEFINED_BY)
```
Shall we rename, IDX_IDP_AUTH_DEFINED_BY 
as there are few tables having AUTH in there name. (Eg: SP_AUTH_SCRIPT, SP_AUTH_STEP etc)

Reply:
Renamed